AsyncDisplayKit/ASTableView.mm

```diff
@@ -47,8 +47,8 @@ @implementation _ASTableViewCell
 
 - (void)layoutSubviews
 {
-  [_delegate willLayoutSubviewsOfTableViewCell:self];
   [super layoutSubviews];
+  [_delegate willLayoutSubviewsOfTableViewCell:self];
```

Comment:
Oh, excellent, I just realized this is in our Cell subclass and is entirely internal.  Let's rename it to -didLayoutSubviewsOfTableViewCell:, and we're good to land!  Really thrilled you investigated this issue right on the heels of writing a detailed report - thank you, @aaronschubert0!